### DISCUSSION TITLE
今天Docker更新后，版本反而后退了。

### DISCUSSION THREAD

**Discussion by Artyaya** [AlistGo/alist#4158] [General]
用 xhofe/alist:latest 拉取到的镜像是 3.10；
我取消了所有加速镜像站点，再拉一次，还是3.10；
前天拉取的是3.15。

请老大检查一下，谢谢。

**Comment by fun-li** [comment]
想问问docker拉取最新的镜像后怎么恢复之前的配置？

**Reply by zch11230** [reply]
> 想问问docker拉取最新的镜像后怎么恢复之前的配置？

1.拉取新镜像；
2.停止旧容器，修改原来容器的网络端口为自动；
3.复制原来的容器，修改复制出来的容器端口；
4.修改两个容器的名字，启动测试新容器正常后删除旧容器。

问题是现在docker登不上啊，你们怎么是怎么玩的。

**Reply by fun-li** [reply]
好像把旧的容器删掉后，拉取最新的镜像直接可以保留配置了，我昨天下午试了一下，是可以的，至于你说的docker登不上指的docker hub么，用梯子是可以的
